src/zenml/models/v2/core/step_run.py

```diff
@@ -314,6 +313,15 @@ class StepRunResponseMetadata(ProjectScopedResponseMetadata):
 class StepRunResponseResources(ProjectScopedResponseResources):
     """Class for all resource models associated with the step run entity."""
 
+    logs: Optional["LogsResponse"] = Field(
+        title="Logs associated with this step run.",
+        default=None,
+    )
```

Comment:
I somehow expected this to break the UI, but it didn't. Why move them from the metadata in the first place ?